README.md

```diff
@@ -192,13 +195,21 @@ The backtrace-unity library includes support for capturing Android NDK crashes a
 
 `system.memory` usage related information including memfree, swapfree, and vmalloc.used is now available. Additional VM details and voluntary / nonvountary ctxt switches are included.
 
-## ANRs and Hangs
+## ANRs and Hangs <a name="anr-reporting"></a>
+
+When configuring the backtrace-unity client for an Android deployment, programmers will have a toggle available in the backtrace-unity GUI in the Unity Editor to enable or disable ANR or Hang reports. This will use the default of 5 seconds. This will be made configurable in a future release. 
+
+The `error.type` for these reports will be `Hang`.
+
+## (Early Access) Low Memory Reports <a name="oom-reporting"></a>
+
+Backtrace can detect low memory situations for a game running in Unity on Android devices, and attempt to generate an error report with an associated dump object for further investigation. When configuring the backtrace-unity client for an Android deployment, programmers will have a toggle available in the backtrace-unity GUI in the Unity Editor to enable or disable sending Low memory warnings to Backtrace. 
 
-When configuring the backtrace-unity client for an Android deployment, programmers will have a toggle available in the backtrace-unity GUI in the Unity Editor to enable or disable ANR or Hang reports. This will use the default of 5 seconds. The `error.type` for these reports will be `Hang`.
+The `error.type` for these reports will be `Low Memory`.
 
-## Low Memory Reports (Early Access)
+Our integration is built in such a way that if multiple low memory warnings are being generated in rapid succession, we will only send a report every 2 minutes. Currently, this is not configurable.
```

Comment:
we're not sending report every 2 minutes. We're regenerating memory warning data in application state cache. So when applicaiton crashed because of oom exception on the next applicaiton session we will have the latest available version of memory warning.

Reply:
Sorry @konraddysput I missed this. I don't understand what the 2 minutes timer does then. Can you elaborate?

Reply:
Mobile games can generate a lot low memory warnings. In this case on iOS when we receive a memory warning, we should be smarter on generating low memory report.  So when the game received a new memory warning, but we already stored memory warning in the application cache, we don't need to generate it once again. 2 minutes means that we will generate a new application state when previous memory warning is older than 2 minutes.